tests/AttributeBagTest.php

```
@@ -0,0 +1,149 @@
+        );
+    }
+
+    #[DataProviderExternal(AttributeBagProvider::class, 'invalidKey')]
+    public function testGetThrowsForInvalidKey(string|UnitEnum $key): void
+    {
+        $attributes = ['id' => 'submit'];
+
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(Message::KEY_MUST_BE_NON_EMPTY_STRING->getMessage());
+
+        AttributeBag::get($attributes, $key);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'testGetThrowsForInvalidKey'. (undefined)

(StaticAccess)